### PR TITLE
Expense Tracker

### PR DESCRIPTION
**Pull request by Saimanjari777**
### Issue Title:  Expense Tracker

- **Name of the project** : Expense Tracker
- **Category:** Vanilla JS
- **Name:** Sai Manjari
- **GitHub ID:** https://github.com/Saimanjari777
- **Idenitfy yourself:** GSSiC

Closes: #239 

### Describe the add-ons or changes you've made 📃

II have added expense tracker where we can add, delete and manage expenses. It also shows the total amount spent.

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### How Has This Been Tested? ⚙️

I have tested this on my local computer

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Note to reviewers 📄

Please review and specify if any changes required.

### CODE REVIEW

File: Vanilla-JS-Projects/Basic/Expense-Tracker/index.html
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/Expense-Tracker/index.html:1]: format the code properly.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Saimanjari777 format the code files `index.html`, `script.js` and `style.css`.

**Comment by Saimanjari777** [comment]
I have made the changes, Please check @Avdhesh-Varshney

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Saimanjari777 how this is formatted? you have just removed comments only. See there are some code are not aligned properly. So, simply format the code using vs-code by click right click and search for the option of `Format Code`.
image

**Comment by Saimanjari777** [comment]
Done @Avdhesh-Varshney

**Review by Avdhesh-Varshney — APPROVED** [review]
@Saimanjari777 PR Approved 🎉

**Comment by Avdhesh-Varshney** [comment]
@Saimanjari777 carefully link your issue with the pr.

**Pull request merged**